portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,102 @@
 // Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-/**
- * Adds a random greeting to the page.
- */
+
+let elementsToShow;
+let scroll;
+
+$(document).ready(function() {
+    scroll = window.requestAnimationFrame || function(callback){ window.setTimeout(callback, 1000/60)};
+    elementsToShow = document.querySelectorAll('.animate');    
+    loop();
+    asyncRemovePlaceHolder();
+})
+
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    revertToUnchecked();
+
+    const greetings =
+        ["\"In a world too often governed by corruption and arrogance, it can be difficult to stay true to one's philosophical and literary principles.\"   -Lemony Snicket, A Series of Unfortunate Events", "\"Did you know that the Sparrow flies south for winter?\"   -Skulduggery Pleasant, Skulduggery Pleasant", "\"Someday in your life, you'll realize there is no turning back. There was never any turning back.\"   -u/JayStar1213"];
```

Comment:
you can define it as a const outside the class instead of everytime you call the function

Reply:
changed arrays to have global scope.